Documentation/project-docs/library-servicing.md

```diff
@@ -0,0 +1,33 @@
+# How to service a library in CoreFx
+
+This document provides the steps necessary after modifying a CoreFx library in a servicing branch (where "servicing branch" refers to any branch whose name begins with `release/`).
+
+## Check for existence of a .pkgproj
+
+Most CoreFx libraries are not packaged by default. Some libraries have their output packaged in `Microsoft.Private.CoreFx.NetCoreApp`, which is always built, while other libraries have their own specific packages, which are only built on-demand. Your first step is to determine whether or not your library has its own package. To do this, go into the root folder for the library you've made changes to. If there is a `pkg` folder there (which should have a `.pkgproj` file inside of it), your library does have its own package. If there is no `pkg` folder there, the library is built as part of `Microsoft.Private.CoreFx.NetCoreApp` and shipped as part of `Microsoft.NetCore.App`. If this is the case, there is nothing that needs to be done.
+
+For example, if you made changes to [System.Data.SqlClient](https://github.com/dotnet/corefx/tree/5da5fd3628253da1d2578ab5c6a437202ac76254/src/System.Data.SqlClient), then you have a .pkgproj, and will have to follow the steps in this document. However, if you made changes to [System.Collections](https://github.com/dotnet/corefx/tree/5da5fd3628253da1d2578ab5c6a437202ac76254/src/System.Collections), then you don't have a .pkgproj, and you do not need to do any further work for servicing.
+
+## Determine PackageVersion
+
+Each package has a property called `PackageVersion`. When you make a change to a library & ship it, the `PackageVersion` must be bumped. This property could either in one of two places inside your library's source folder: `Directory.Build.Props` (called `dir.props` in 2.0.0 & earlier), or the `.pkgproj`. It's also possible that the property is in neither of those files, in which case we're using the default version from [Packaging.props](https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/Packaging.props#L36) (IMPORTANT - make sure to check the default version from the branch that you're making changes to, not from master). You'll need to increment this package version. If the property is already present in your library's source folder, just increment the patch version by 1 (e.g. `4.6.0` -> `4.6.1`). If it's not present, add it to `Directory.Build.props`, where it is equal to the version in `Packaging.props`, with the patch version incremented by 1. 
```

Comment:
NIT:  
> If it's not present, add it to `Directory.Build.props`

Change it to `If it is not present, add it to the library's Directory.Build.props` so that we are more explicit about which Directory.Build.props we are talking about (there are too many different ones across the repo.